src/revChatGPT/V1.py

```diff
@@ -264,7 +264,7 @@ def ask(
             try:
                 line = json.loads(line)
             except json.decoder.JSONDecodeError:
-                log.info("Error parsing JSON")
+                log.error("Error parsing JSON")
```

Comment:
This isn't a true error. Happens often due to bad escapes by OpenAI. Has to be info

Reply:
And can just be passed since the next line would have better escapes

Reply:
I see, so this should also be `debug`  or be hiden for upper caller?

Reply:
I suppose just remove logging for that in general. It doesn't do much